cmd/protecodeExecuteScan.go

```diff
@@ -376,7 +376,7 @@ func uploadFile(utils protecodeUtils, config protecodeExecuteScanOptions, produc
 			combinedFileName = fmt.Sprintf("%v_%v", config.PullRequestName, fileName)
 		}
 
-		resultData := client.UploadScanFile(config.CleanupMode, config.Group, pathToFile, combinedFileName, version, productID, replaceBinary)
+		resultData := client.UploadScanFile(config.CleanupMode, config.Group, config.CustomDataJSONMap, pathToFile, combinedFileName, version, productID, replaceBinary)
```

Comment:
`client.DeclareFetchURL()` also needs to be modified, missing `config.CustomDataJSONMap` input, since _/api/fetch/_ headers also accepts custom data headers.

Reply:
Done 👍